RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -113,6 +114,7 @@
    chargedHadronTrackResolutionFilter *
    BadChargedCandidateFilter*
    BadPFMuonFilter *
+   hfNoisyHitsFilter *
```

Comment:
this should be added only for run2_miniAOD_UL to more properly follow the no-change requirement/policy

Reply:
In fact the list of filters here is outdated anyway so I simply dropped that line. My understanding is that it's better to update this in separate PR (both for 106X and master).

Reply:
> In fact the list of filters here is outdated anyway so I simply dropped that line. My understanding is that it's better to update this in separate PR (both for 106X and master).

I'm guessing that the update was made in a wrong place.
- my comment was about the `metFilters = cms.Sequence(`, which defines results of a path `Flag_METFilters = cms.Path(metFilters)`. This part still needs an update to be protected with `run2_miniAOD_UL`. If the combined Flag_METFilters is outdated, it's fine to remove `hfNoisyHitsFilter` here for good, but then it better be also removed in the master as well.
- your update was in `metFilterPathsTask`, which is used in `slimmingTask` in case someone is to consume the bool tags produced by one of the modules. At this point (nothing is consuming this bool, AFAIK) it's a mostly technical update, but it would still need a match in the master first before this 106X can proceed.